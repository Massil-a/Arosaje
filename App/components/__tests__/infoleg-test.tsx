import React from 'react';
import { render } from '@testing-library/react-native';
import InfoLeg from '../../app/optnav/infoleg';

describe('InfoLeg', () => {
    it('renders correctly', () => {
        const { toJSON } = render(<InfoLeg />);
        expect(toJSON()).toMatchSnapshot();
    });

    it('displays the correct title', () => {
        const { getByText } = render(<InfoLeg />);
        expect(getByText('Informations Légales')).toBeTruthy();
    });

    it('displays the legal information text', () => {
        const { getByText } = render(<InfoLeg />);
        expect(getByText(/Ceci est un exemple d'informations légales pour votre application de garde de plante./)).toBeTruthy();
        expect(getByText(/Veuillez consulter un professionnel pour obtenir des informations légales précises et adaptées à votre situation./)).toBeTruthy();
    });
});
