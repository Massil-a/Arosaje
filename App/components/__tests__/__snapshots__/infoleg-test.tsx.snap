// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoLeg renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#f9f9f9",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "padding": 20,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 18,
          "fontWeight": "bold",
          "marginBottom": 10,
        }
      }
    >
      Informations Légales
    </Text>
    <Text
      style={
        {
          "fontSize": 14,
        }
      }
    >
      Ceci est un exemple d'informations légales pour votre application de garde de plante. Veuillez consulter un professionnel pour obtenir des informations légales précises et adaptées à votre situation.
    </Text>
  </View>
</View>
`;
